docs/semgrep-app/findings.md

```diff
@@ -95,42 +95,72 @@ The Findings page consists of:
     ![Findings information panel](/img/app-findings.png)<br />
     *Figure 2.* Findings information panel.
 
-## Triaging findings
-
-**Triaging** means prioritizing a finding based on a policy or criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
-
-Semgrep App uses the logic specified in the table below to automatically mark findings as either fixed or removed when a finding is no longer present in the code. You can also ignore findings in Semgrep App directly through **triage** or **bulk triage**.
-
-The triage statuses are as follows:
-
-<TriageStatuses />
-
 ### Filtering findings
 
 Filtering allows you to easily isolate groups of findings for ease in triaging and identifying related groups of issues. The following criteria are available for filtering:
 
 | Filter      | Description  |
 | ----------  | ------------ |
-| **Status**       | Filter for different findings triage states. Refer to the [following table](#triaging-findings) to understand triage states. |
 | **Projects**     | Filter by repositories connected to Semgrep App. |
-| **Branches**     | Filter by findings in different branches. |
-| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. |
-| **Rulesets**     | Filter by name of the ruleset where rules that matched the code belong. |
+| **Status**       | Filter for different findings triage states. Refer to the [following table](#triaging-findings) to understand triage states. |
+| **Severities**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
 | **Actions**     | Filter by monitoring, commenting, or blocking rules in your Rule Board. |
-| **Severity**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
+| **Confidences** | Filter by indication of the rule to detect true positives. The higher the confidence the more true positives the rule may detect.  |
+| **Categories**  |  Filter by various rule categories, such as security, or best practice rules. More than one category can be selected for filtering. |
+| **Branches**     | Filter by findings in different Git branches. More than one branch can be selected for filtering. |
+| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. More than one rule can be selected for filtering. |
```

Comment:
I think we repeated "More than one rule can be selected for filtering" twice here

Reply:
```suggestion
| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. |
```